homework/length-sort/sort.cpp

```diff
@@ -0,0 +1,20 @@
+#include "sort.hpp"
+#include <algorithm>
+#include <deque>
+#include <forward_list>
+#include <string>
+
+bool isEqualSize(const std::string& lhs, const std::string& rhs) {
+    if (lhs.size() == rhs.size()) {
+        return std::lexicographical_compare(lhs.begin(), lhs.end(), rhs.begin(), rhs.end());
```

Comment:
did you know that compare to string is done lexicographical :)?
Sou you can use lhs == rhs